api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1232 @@
+            reached, the messages are immediately flushed to the client.
+
+            * If 0 or undefined, will assume 30 seconds.
+          type: integer
+          format: int32
+          required: false
+          default: 30
+        - name: stream_timeout
+          in: query
+          description: |
+            Maximum time in seconds a stream will live before being interrupted.
+            If unspecified will stream indefinately.
+
+            If this timeout is reached, any pending messages (in the sense of `stream_limit`) will be flushed
+            to the client.
+
+            Stream initialization will fail if `stream_timeout` is lower than
+            `batch_flush_timeout`.
+          type: integer
+          format: int32
+          required: false
+        - name: batch_keep_alive_limit
+          in: query
+          description: |
+            Maximum number of keep-alive messages to get in a row before closing the connection. Unlimited by
+            default.
+          type: integer
+          format: int32
+          required: false
+        - name: X-Flow-Id
+          in: header
+          description: |
+            The flow id of the request, which is written into the logs and passed to called services. Helpful
+            for operational troubleshooting and log analysis.
+          type: string
+          format: flow-id
+      responses:
+        '200':
+          description: |
+            Starts streaming to the client.
+            Stream format is a continuous series of `EventStreamBatch`s separated by `\n`
+          schema:
+            $ref: '#/definitions/EventStreamBatch'
+        '400':
+          description: Bad syntax
+          schema:
+            $ref: '#/definitions/Problem'
+        '401':
+          description: Not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: |
+            Topic not found.
+          schema:
+            $ref: '#/definitions/Problem'
+        '422':
+          description: Unprocessable entity
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Internal Server Error. Details are provided on the returned `Problem`.
+          schema:
+            $ref: '#/definitions/Problem'
+    post:
+      tags:
+        - topic-api
+        - draft
+      description: |
+        Post one `Event`s to a given `Topic` (of an `EventType`). Upon reception the `EventType`'s
+        validation and enrichment rules are applied. For details on their behaviour please see POST /even
+        t-types/{name}/events.
+
+        (TBD: submission of batches of events)
+
+        Restriction: To avoid inconsistencies in the relative ordering of Events, **this operation cannot be
+        used if this Topic's defining EventType defines a `PartitionResolutionStrategy`.** Doing so will
+        produce a 405 Not allowed.
```

Comment:
editorial: it's worth to hint that that post request repetition creates event duplicates.